demo/napari_gui/napari_.py

```diff
@@ -0,0 +1,26 @@
+# load data
+from skimage.io import imread
+image = imread('https://samples.fiji.sc/blobs.png')
+
+# start up napari
+import napari
+import pyclesperanto_prototype as cle
+with napari.gui_qt():
+    viewer = napari.Viewer()
+    viewer.add_image(image, name='blobs')
+
+    # push image to GPU
+    input = cle.push_zyx(image)
+
+    # process the mage
+    sigma = 3
+    blurred = cle.gaussian_blur(input, sigma_x=sigma, sigma_y=sigma)
+    binary = cle.threshold_otsu(blurred)
+    labels = cle.connected_components_labeling_box(binary)
+
+    # pull result back
+    output = cle.pull_zyx(labels)
```

Comment:
Can this step be skipped because of `__array__`?

Reply:
likely yes. I'd like to keep it for the moment to be consistent with the other languages/dialects. But I strongly agree with @royerloic - it would be awesome to make clesperanto more transparent and numpy-like. Push and pull might fall first when implementing this.
See:
https://twitter.com/loicaroyer/status/1331712368024272897?s=20
and
https://github.com/clEsperanto/pyclesperanto_prototype/issues/33